assets/quick-order-form.css

```diff
@@ -34,10 +34,15 @@
 }
```

Comment:
Should this be highlighted too? cc @melissaperreault 
https://screenshot.click/11-52-dgocv-ulncg.png

Reply:
Oh yes! good find!

Reply:
fixed here [c422aa7](https://github.com/Shopify/dawn/pull/3142/commits/c422aa7a2be36057fb5a95490333cf686d06760a)

Reply:
👋 @eugenekasimov , I think we can achieve this with `:has`

Video: https://screenshot.click/21-44-97rj6-cbpo7.mp4

Docs: https://developer.mozilla.org/en-US/docs/Web/CSS/:has

This way we dont need JS

Reply:
We already have `has` in the theme code in Dawn so I think this is ok 👍

Reply:
Changed the approach here [2a0c648](https://github.com/Shopify/dawn/pull/3142/commits/2a0c6489f7db373c97c504ee931cb3a13a076ff7) 👍